scripts/build_test.py

```diff
@@ -24,7 +33,79 @@
 class BuildTests(test_utils.GenericTestBase):
     """Test the build methods."""
 
+    def test_minify(self):
+        """Tests _minify with an invalid filepath."""
+        # pylint: disable=protected-access
+        with self.assertRaises(subprocess.CalledProcessError) as calledProcess:
+            build._minify(
+                'path/to/input.js', 'path/to/output.js')
```

Comment:
Perhaps include the string "invalid" in the invalid filepath

Reply:
Done!